nextflow.config

```diff
@@ -81,12 +81,14 @@ params {
     save_runmerged_reads        = false
 
     // Host Removal
-    perform_shortread_hostremoval     = false
-    shortread_hostremoval_reference   = null
-    shortread_hostremoval_index       = null
-    save_hostremoval_index            = false
-    save_hostremoval_mapped           = false
-    save_hostremoval_unmapped         = false
+    perform_shortread_hostremoval          = false
+    shortread_hostremoval_reference        = null
+    shortread_hostremoval_index            = null
+    longread_hostremoval_index             = null
```

Comment:
Are we missing the `longread_hostremoval_reference` here? Or do we use the same? If so should make the `hostremoval_reference` generic I guess?

Reply:
In the code I've taken `shortread_hostremoval_reference` as the same reference for long read host removal. I will change `shortread_hostremoval_reference` to `hostremoval_reference`.

I noticed also that `perform_longread_hostremoval          = false` was missing so I will add that there too.